App/Sagas/HandleImageUploadComplete.ts

```diff
@@ -0,0 +1,38 @@
+import { call, put, select } from 'redux-saga/effects'
+import { ActionType } from 'typesafe-actions'
+import RNFS from 'react-native-fs'
+
+import TextileNode from '../../TextileNode'
+import ProcessingImagesActions, { ProcessingImagesSelectors, ProcessingImage } from '../Redux/ProcessingImagesRedux'
+import getDefaultThread from './GetDefaultThread'
+import { Thread } from '../Models/TextileTypes'
+
+export function * handleImageUploadComplete (action: ActionType<typeof ProcessingImagesActions.imageUploadComplete>) {
```

Comment:
Step #2. Once the upload is complete, this guy adds the photo to the wallet and shared it to the thread the user originally chose.